config/kubeadm/patches/kubeadm_client_config_validating_webhook.yaml

```diff
@@ -0,0 +1,21 @@
+- op: add
+  path: "/webhooks/-"
+  value:
+    clientConfig:
+      caBundle: Cg==
+      service:
+        name: webhook-service
+        namespace: system
+        path: /validate-cluster-x-k8s-io-v1alpha3-kubeadmcontrolplane
+    failurePolicy: Fail
+    name: validation.kubeadmcontrolplane.cluster.x-k8s.io
+    rules:
+    - apiGroups:
+      - cluster.x-k8s.io
+      apiVersions:
+      - v1alpha3
+      operations:
+      - CREATE
+      - UPDATE
+      resources:
+      - kubeadmcontrolplanes
```

Comment:
best to add the missing new lines at the end of new files.

Reply:
thx. fixed in amended commit.